BankingSystem/ApplicationDbContext.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.EntityFrameworkCore;
+using BankingSystem.Models;
+
+namespace BankingSystem
+{
+    public class ApplicationDbContext : DbContext
+    {
+        public ApplicationDbContext(DbContextOptions dbContextOptions) : base(dbContextOptions) { }
+
+        public DbSet<AccountType> AccountTypes { get; set; }
+        public DbSet<PaymentMethod> PaymentMethods { get; set; }
+        public DbSet<BankAccount> BankAccounts { get; set; }
+        public DbSet<BankTransaction> BankTransactions { get; set; }
+        public DbSet<BankAccountPosting> BankAccountPostings { get; set; }
+
+
+        // applying seed data to the database as part of migration
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<AccountType>().HasData(
```

Comment:
Use IEntityTypeConfiguration interface for specifying configs and seeding data of each entity